src/Fixer/ControlStructure/NoUselessElseFixer.php

```diff
@@ -57,18 +59,24 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
     /**
      * {@inheritdoc}
      */
-    public function getPriority()
+    public function getDefinition()
     {
-        // should be run before NoWhitespaceInBlankLineFixer, NoExtraConsecutiveBlankLinesFixer, BracesFixer and after NoEmptyStatementFixer.
-        return 25;
+        return new FixerDefinition(
+            'There should not be useless else cases.',
+            null,
+            array(
+                new CodeSample("<?php\nif (\$a) {\n    return 1;\n} else {\n    return 2;\n}"),
```

Comment:
don't hesitate to use `'` instead of `"` and real line endings instead of `\n` when it would be more readable ;)